Makefile

```diff
@@ -79,14 +79,16 @@ pylint:
 	@pylint --disable=W0511 -s n extras/scripts/gen_manifest.py
 	@pylint --disable=R0902 -s n cli/kubectl_kadalu/storage_list.py
 	@pylint --disable=R0902 -s n cli/kubectl_kadalu/utils.py
+	@pylint --disable=R0801 -s n cli/kubectl_kadalu/storage_add.py
+	@pylint --disable=R0801 -s n cli/kubectl_kadalu/storage_remove.py
```

Comment:
this is not needed.. as  L91 does the same..

Reply:
removed